source/common/router/config_impl.cc

```diff
@@ -332,8 +332,6 @@ RouteEntryImplBase::RouteEntryImplBase(const VirtualHostImpl& vhost,
                          : ""),
       path_redirect_(route.redirect().path_redirect()),
       path_redirect_has_query_(path_redirect_.find('?') != absl::string_view::npos),
-      enable_preserve_query_in_path_redirects_(Runtime::runtimeFeatureEnabled(
```

Comment:
One last comment -- I think you can remove `#include "common/runtime/runtime_features.h"`

Reply:
oops sorry about force pushing... I have removed this line.